NewForce-at-Mountwest/bangazon-site-take-2-centaurs
# Ticket 1

DESCRIPTION

- User cannot see affordance for Sell Something when they're not logged in/authenticated

- Once user logs in, user may see and click on Sell Something

- When the user fills out all fields of the form to list an item to sell, the user may click the sell button. The user will then be redirected to the details page for that product.

- If the user doesn't enter the proper information into the required fields, the user will receive an error message.

RELATED TICKETS

- Ticket # 1 (https://trello.com/c/XoUtxKAG/1-1-customer-can-add-a-product-to-sell)

STEPS TO TEST

git add and git commit what you're working on on your branch
git checkout master to switch to your local master branch
git fetch --all will give you access to your teammate's branches (if they've pushed them up)
git checkout na-add-products to access my branch.
If you don't see the most recent version of their files, try git pull origin na-add-products on your server on my branch and make sure everything looks okay.
When you're not logged in, you shouldn't see the 'Sell Something' affordance
Once you're logged in, use your navbar and click on the 'Sell Something' affordance
Complete the form
If you enter invalid information into the required fields, such as more than 55 characters in the title field, you should receive an error message
If you complete the form with valid information and submit the form, you'll be redirected to the details of the product you just added
Profit.


Checklist:
[] I have commented my code, particularly in hard-to-understand areas
[] I have made corresponding changes to the documentation
[]My changes generate no new errors
[] I have added tests that prove my fix is effective or that my feature works
[] New and existing unit tests pass locally with my changes
